listenbrainz/webserver/rabbitmq_connection.py

```diff
@@ -1,25 +1,39 @@
 import sys
 from time import sleep
-import pika
+from flask import Flask
+from flask.ext.pika import Pika as FPika
 
 _rabbitmq = None
 
 def init_rabbitmq_connection(app):
     """Create a connection to the RabbitMQ server."""
     global _rabbitmq
 
-    app.logger.info("Call to init_rabbit_mq_connection")
     if "RABBITMQ_HOST" not in app.config:
         app.logger.error("RabbitMQ host:port not defined. Sleeping 2 seconds, and exiting.")
         sleep(2)
         sys.exit(-1)
 
+    FLASK_PIKA_PARAMS = {
```

Comment:
Normally it should be in config.py, but that is already getting cluttered. When I work on deployment I'll see about adding production config for the pooling settings.